src/dlr_relayvm.cc

```diff
@@ -75,6 +75,13 @@ void RelayVMModel::SetupVMModule(const std::vector<DLRModelElem>& model_elems) {
 
   LoadJsonFromString(metadata_data, this->metadata_);
   ValidateDeviceTypeIfExists();
+  // Override allocator.
+  const char* val = std::getenv("DLR_VM_ALLOCATOR");
```

Comment:
Looks like this is applicable only for TVM models? So, `DLR_TVM_ALLOCATOR` ?

Reply:
Renamed to `DLR_RELAYVM_ALLOCATOR` because this only applied to relayvm DLR backend.